bot/bot.js

```diff
@@ -1,23 +1,38 @@
 import { Action, MessageType, MapUtility } from '../src/index.js';
 
-export const BOT_NAME = 'Power Mad';
+export const BOT_NAME = 'Turner';
 
-/*
-  This bot is hungry for power. It always goes for the closest power up.
-  It's smart enough to not crash into obstacles, but not smart enough to find a way around them.
-  If there are no power ups, it just moves in a random valid direction.
-*/
+const directionActions = [Action.Up, Action.Right, Action.Down, Action.Left];
 
-/**
- * @template T
- * @param {readonly T[]} items
- * @returns {T}
- */
-function randomItem(items) {
-  return items[Math.floor(Math.random() * items.length)];
+var currentDirection = 0;
```

Comment:
Don't use var
```suggestion
let currentDirection = 0;
```

Reply:
You should reset the bot state between games, check the comment in the onMessage handler

Reply:
You could also create a bot class and use a new instance for each game